### DISCUSSION TITLE
Refactor and clean up for the example apps

### DISCUSSION THREAD

**Discussion by emadbaqeri** [functionland/box#233] [Ideas]
Following issue  #232 some changes need to be applied to the example apps. @matiqorb had a great about the new structure of the sample projects that require more thoughts on that. @matiqorb please let us know about the idea over here at the discussions

**Comment by ghorbani-m** [comment]
The idea is to haveing a single project with multi-samples folders on it, each simple sample should be written in a single file. if we need a complex sample we can separate it into multiple files in a single folder to have a clean code.
Finally, to manage all samples, we can create a starter page to display all available sample names like a tile box on this page, then users could navigate between all samples easily.

![Sample starter](https://user-images.githubusercontent.com/16715868/175761172-508480ca-c411-4e88-82fe-b85c24375380.png)
